pkg/bmc/redfish.go

```diff
@@ -38,6 +39,12 @@ func newRedfishiDracVirtualMediaAccessDetails(parsedURL *url.URL) (AccessDetails
 	}, nil
 }
 
+func newRedfishiDracAccessDetails(parsedURL *url.URL) (AccessDetails, error) {
```

Comment:
Tests are failing because #396 merged, so this also needs to take a bool for `disableCertificateVerification`.